packages/components/src/components/date-picker/date-picker-native.tsx

```diff
@@ -84,7 +106,7 @@ const Native = ({
                 type='date'
                 max={max_date && toMoment(max_date).format('YYYY-MM-DD')}
                 min={min_date && toMoment(min_date).format('YYYY-MM-DD')}
-                error={error}
+                // error={error}
```

Comment:
remove the commented code

Reply:
removed this line, since there is no such property `error` for html input element